images/dcos/root/etc/nginx/conf.d/default.conf.template

```diff
@@ -1,5 +1,13 @@
 server {
-    listen <PORT>;
+    listen <PORT> ssl spdy;
+    ssl_certificate snakeoil.crt;
```

Comment:
would be nice to allow the user to override via config-prop/envvar's like `kubernetes.certificate.public` and `kubernetes.certificate.private`, so as to avoid the checked-in (thereby insecure) defaults. it wouldn't hurt the size of our bootstrap too much to incorporate a few envvar checks. i think the marathon-lb project uses envvars for this. https://github.com/mesosphere/marathon-lb#providing-ssl-certificates